src/core/battle/execute.rs

```diff
@@ -673,13 +673,15 @@ impl ExecuteContext {
 fn execute_use_ability_knockback(
     state: &mut State,
     command: &command::UseAbility,
+    _: ability::Knockback,
 ) -> ExecuteContext {
     let mut context = ExecuteContext::default();
     let id = state::blocker_id_at(state, command.pos);
     let from = command.pos;
     let actor_pos = state.parts().pos.get(command.id).0;
     let dir = Dir::get_dir_from_to(actor_pos, command.pos);
     let to = Dir::get_neighbor_pos(command.pos, dir);
+    // Knockback strength might be used to push farther than 1 tile ?
```

Comment:
I'm unsure if strength would mean :
- "distance to push" : strength of 1: pushes 1 tile ; strength of 3: pushes 3 tiles
- or "difficulty to resist": strength of 1 can be resisted by "Heavy" blockers ; strength of 3 cannot.

I assumed the first approach was preferred, without implementing different distances.

Reply:
I was thinking only about "difficulty to resist" for now.